package.json

```diff
@@ -51,6 +51,7 @@
     "babel-core": "^6.5.2",
     "babel-eslint": "^6.0.4",
     "babel-loader": "^6.2.0",
+    "babel-plugin-__coverage__": "^0.111111.11",
```

Comment:
Yea, I wish this said `"babel-plugin-coverage": "^0.1.1"`.  Some other folks are pushing for a "normal" 1.0 release as well on [this issue](https://github.com/dtinth/babel-plugin-__coverage__/issues/29).